src/app/Main/Footer/Footer-Links.component.tsx

```diff
@@ -68,7 +67,11 @@ export function FooterLinksComponent(): JSX.Element {
                     FoodRisk-Labs
                 </a>
             </li>
-            <li css={footerElementStyle}>{t("Content.Faq")}</li>
+            <li css={footerElementStyle}>
+                <span css={linkStyle}>
```

Comment:
linkStyle is a flex box but used only for styling single elements. Consider removing the flex-box and using a simpler mechanism.